packages/p2p/src/components/recommend-user/recommend-user.tsx

```diff
@@ -66,7 +73,7 @@ const RecommendUser = ({
                 <Button className='recommend-user__block' onClick={handleSelectYes} secondary>
                     <Icon
                         className='recommend-user__block__icon'
-                        color={!is_yes_selected && 'disabled'}
+                        color={!is_yes_selected ? 'disabled' : undefined}
```

Comment:
needed to use a ternary to return undefined as the color prop accepts only string or undefined and not boolean values. on the left it would return false 

![Screenshot 2023-06-01 at 10 40 36 AM](https://github.com/farrah-deriv/deriv-app/assets/103412909/dc1e5fca-1441-413a-9ac0-aeb0278ec163)